pkg/minikube/translate/translations.go-e

```diff
@@ -0,0 +1,408 @@
+// Code generated by go-bindata. (@generated) DO NOT EDIT.
```

Comment:
please remove all of these go-bin data files from your PR those are from previous minikube, we removed the go-bindata